packages/colors-block/src/components/ColorName.tsx

```diff
@@ -30,6 +30,8 @@ export const ColorName = ({ viewType, initialColorName, isEditing, onBlur }: Col
                     value={colorName}
                     onChange={handleColorNameChange}
                     onBlur={onBlur}
+                    draggable={true}
+                    onDragStart={(event) => event.preventDefault()}
```

Comment:
Why?

Reply:
Simone asked that you can select the name and not drag when selecting

Reply:
then just `draggable={false}` no?

Reply:
That doesn't work 🤷‍♂️ Probably the dnd takes precedence, but not sure